test/application_system_test_case.rb

```diff
@@ -150,18 +150,19 @@ def sign_out_for(display_details)
     click_on "Logout"
 
     # make sure we're actually signed out.
-    # (this will vary depending on where you send people when they sign out.)
-    assert_text "Sign In"
+    assert_text "Signed out successfully"
   end
 
   def sign_in_from_homepage_for(display_details)
-    # TODO the tailwind port of bullet train doesn't currently support a homepage.
+    # We actually want to go straight to the user session and
+    # not the home page in case the home page doesn't have new session logic.
     visit new_user_session_path
 
     # this forces capybara to wait until the proper page loads.
     # otherwise our tests will immediately start trying to match things before the page even loads.
     assert_text("Sign In")
   end
+  alias_method :new_session_page_for, :sign_in_from_homepage_for
```

Comment:
I added the alias method here and the other below because I think it makes more sense naming them this way. I also used an alias method instead of simply re-writing the method in case any developers decided to use this method for their own tests.

Reply:
I like the new names. I wonder if we should use those names for the actual methods and then use the aliases to preserve backwards compatibility (for now). Or maybe instead of just a raw `alias_method` we implement an actual method that would show a deprecation warning and then call the original method?

Something like:

```ruby
def sign_in_from_homepage_for(display_details)
  puts "sign_in_from_homepage_for is deprecated".red
  puts "  please switch to: new_session_page_for".red
  puts "  called from #{caller.first}".red
  new_session_page_for(display_details)
end
```